manifest

```diff
@@ -4,3 +4,6 @@ repository=elixir
 parent=base
 variants=(node browsers)
 namespace=cimg
+parentTags=(20.3 21.3 22.3 23.3 24.3)
```

Comment:
Erlang versions before v22 isn't supported for the current/modern versions of Elixir. https://hexdocs.pm/elixir/1.13.4/compatibility-and-deprecations.html

Reply:
i'll close out #73 and include those changes here instead. I agree with respinning so users can use the feature right away given there are a couple issues linked

got it - i was referencing the OTP compatibility since we had older versions in the repo. i'll remove versions <22